Neos.Flow/Classes/Security/Authentication/Token/BearerToken.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Flow\Security\Authentication\Token;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Mvc\ActionRequest;
+use Neos\Flow\Security\Authentication\TokenInterface;
+use Neos\Flow\Security\Exception\AuthenticationRequiredException;
+use Neos\Flow\Security\Exception\InvalidAuthenticationStatusException;
+
+/**
+ * rfc6750 Bearer token (https://tools.ietf.org/html/rfc6750)
+ *
+ * "A security token with the property that any party in possession of the token (a "bearer")
+ * can use the token in any way that any other party in possession of it can. Using a bearer
+ * token does not require a bearer to prove possession of cryptographic key material (proof-of-possession)."
+ */
+class BearerToken extends AbstractToken implements SessionlessTokenInterface
+{
+
+    /**
+     * The password credentials
+     * @var array
+     * @Flow\Transient
+     */
+    protected $credentials = ['bearer' => ''];
+
+    /**
+     * @param ActionRequest $actionRequest
+     * @throws AuthenticationRequiredException
+     * @throws InvalidAuthenticationStatusException
+     */
+    public function updateCredentials(ActionRequest $actionRequest)
+    {
+        $this->setAuthenticationStatus(self::AUTHENTICATION_NEEDED);
+        $httpRequest = $actionRequest->getHttpRequest();
+
+        if (!$httpRequest->hasHeader('Authorization')) {
+            return;
+        }
+
+        $authorizationHeader = $httpRequest->getHeader('Authorization')[0];
+
+        if (!str_starts_with($authorizationHeader, 'Bearer ')) {
```

Comment:
https://www.php.net/manual/de/function.str-starts-with.php PHP8+ only - unfortunately not quite there yet :)
```suggestion
        if (substr($authorizationHeader, 0,  7) !== 'Bearer ') {
```

Reply:
Duh, that tells me: We need at least a basic linter test in CI, to catch such issues…

Reply:
Or a (unit) test for every new code ;)

Reply:
Well, this is on purpose. The method is provided by symfony/polyfill-php80 which is required through symfony/console.

But yeah - it is not a direct dependency of Flow.

Reply:
Ah yes, the polyfill - well, feel free to make it a (direct) dependency of flow and we can keep this line as is I'd say :)

Reply:
lol, committed your fix just in this moment ;) Thanks a lot @albe!!

Reply:
Haha, there is the linting 😛 , Ill take care of the style issue